### PR TITLE
Emmc sd.

### PR DESCRIPTION
**Pull request by ArthurSonzogni**
Hi,
I made two improvement to your branch.

Could you please take a look?

### CODE REVIEW

**Comment by SAtacker** [comment]
Am I missing something?
```sh
/home/satacker/Desktop/beagle-config/src/ui/panel/emmc/emmc_impl.cpp: In function ‘std::__cxx11::wstring ui::{anonymous}::Format(float, ui::{anonymous}::sizeApprox)’:
/home/satacker/Desktop/beagle-config/src/ui/panel/emmc/emmc_impl.cpp:25:28: error: ‘setprecision’ is not a member of ‘std’
   ss << std::fixed << std::setprecision(2) << (value / float(unit));
                            ^~~~~~~~~~~~
/home/satacker/Desktop/beagle-config/src/ui/panel/emmc/emmc_impl.cpp:25:28: note: suggested alternative: ‘set_union’
   ss << std::fixed << std::setprecision(2) << (value / float(unit));
                            ^~~~~~~~~~~~
                            set_union
/home/satacker/Desktop/beagle-config/src/ui/panel/emmc/emmc_impl.cpp:35:1: error: control reaches end of non-void function [-Werror=return-type]
 }
 ^
cc1plus: all warnings being treated as errors
make[2]: *** [CMakeFiles/beagle-config.dir/build.make:89: CMakeFiles/beagle-config.dir/src/ui/panel/emmc/emmc_impl.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:284: CMakeFiles/beagle-config.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
```

File: src/ui/panel/emmc/emmc_impl.cpp [hunk 1]
```diff
@@ -32,79 +49,70 @@ class EMMCImpl : public PanelBase {
...
+    {
+      auto currentUserPath = std::string(getpwuid(geteuid())->pw_dir);
+      auto free_space = std::filesystem::space(currentUserPath).free;
+      auto cap = std::filesystem::space(currentUserPath).capacity;
+      auto avail = std::filesystem::space(currentUserPath).available;
+      name_list.push_back(text(to_wstring(currentUserPath)));
+      free_list.push_back(text(Format(free_space, unit)));
+      capacity_list.push_back(text(Format(cap, unit)));
+      available_list.push_back(text(Format(avail, unit)));
     }
```
Comment (SAtacker) [inline src/ui/panel/emmc/emmc_impl.cpp:82]: Why do we need this scope?

**Comment by SAtacker** [comment]
Thanks, @ArthurSonzogni ! The table looks sweet. 
A tiny concern that the button is displayed with a bottom edge missing :sweat: 
[asciicast](https://asciinema.org/a/427656)

File: src/ui/panel/emmc/emmc_impl.cpp (hunk 1 above)
Comment (ArthurSonzogni) [reply src/ui/panel/emmc/emmc_impl.cpp]: It was for symmetry with the for loop.

I decided to add a function, called from both location instead.

**Comment by ArthurSonzogni** [comment]
Hey, I addressed all your comments.
I also added  `gauge` to visualize the remaining space left.

**Pull request merged**